### PR TITLE
Fix incorrect ParticleEngine render patch

### PR DESCRIPTION
**Pull request by IMS212**
Fixes solid particles being drawn twice, and without a Frustum check.

### CODE REVIEW

File: patches/net/minecraft/client/renderer/LevelRenderer.java.patch [hunk 1]
```diff
@@ -81,10 +89,12 @@
...
          if (this.targets.particles != null) {
              this.targets.particles = framepass.readsAndWrites(this.targets.particles);
-@@ -679,6 +_,7 @@
+@@ -678,7 +_,8 @@
+                 resourcehandle1.get().copyDepthFrom(resourcehandle.get());
              }
  
-             this.minecraft.particleEngine.render(p_365299_, p_364282_, this.renderBuffers.bufferSource());
+-            this.minecraft.particleEngine.render(p_365299_, p_364282_, this.renderBuffers.bufferSource());
++            this.minecraft.particleEngine.render(p_365299_, p_364282_, this.renderBuffers.bufferSource(), frustum, resourcehandle1 == null ? ParticleRenderType::translucent : type -> true); // Neo: only render translucent particles here in Fast/Fancy
```
Comment (embeddedt) [inline patches/net/minecraft/client/renderer/LevelRenderer.java.patch]: Use the fully qualified name to avoid adding an import

Comment (IMS212) [reply patches/net/minecraft/client/renderer/LevelRenderer.java.patch]: Was already making a commit to replace it with a lambda, is that fine?

Comment (embeddedt) [reply patches/net/minecraft/client/renderer/LevelRenderer.java.patch]: Yep, but you removed the comment with that commit :stuck_out_tongue:

Comment (IMS212) [reply patches/net/minecraft/client/renderer/LevelRenderer.java.patch]: Fixed.

**Comment by IMS212** [comment]
No clue what happened with the gametest, but this should be ready.

**Review by embeddedt — APPROVED** [review]

**Pull request merged**